src/test/ui/rustdoc/check-doc-alias-attr.rs

```diff
@@ -1,16 +1,28 @@
 #![crate_type = "lib"]
 
 #[doc(alias = "foo")] // ok!
+#[doc(alias("bar", "baz"))] // ok!
 pub struct Bar;
 
 #[doc(alias)] //~ ERROR
 #[doc(alias = 0)] //~ ERROR
-#[doc(alias("bar"))] //~ ERROR
 #[doc(alias = "\"")] //~ ERROR
 #[doc(alias = "\n")] //~ ERROR
 #[doc(alias = "
 ")] //~^ ERROR
 #[doc(alias = "\t")] //~ ERROR
 #[doc(alias = " hello")] //~ ERROR
 #[doc(alias = "hello ")] //~ ERROR
+#[doc(alias = "")] //~ ERROR
```

Comment:
```suggestion
#[doc(alias = "")] //~ ERROR
#[doc(alias = ["foo", "bar"])] //~ SUGGESTION
```

We should show users the correct way to have multiple aliases.